packages/api/tests/endpoints/test-elasticsearch.js

```diff
@@ -271,6 +271,7 @@ test.serial('Reindex request returns 400 with the expected message when source i
     index: defaultIndexName,
     body: { mappings },
   });
+  t.teardown(() => esClient.indices.delete({ index: defaultIndexName }));
```

Comment:
Doesn't the test essentially do this on line 276?

Reply:
Yeah, that's bad duplication. I will remove.